src/java/org/apache/cassandra/service/pager/AggregationQueryPager.java

```diff
@@ -260,6 +276,18 @@ public final boolean hasNext()
             return next != null;
         }
 
+        private void checkTimeout()
```

Comment:
I guess this can be gone if we just override the timeout when running aggregate query on the coordinator?